lib/nodes/addon/custom-drivers/cluster-drivers/controller.js

```diff
@@ -53,6 +53,18 @@ export default Controller.extend({
     },
   },
 
+  bulkActionHandler: computed(function() {
+    return {
+      promptDeactivate: (models) => {
+        get(this, 'modalService').toggleModal('modal-confirm-deactivate', {
+          originalModel: models[0],
+          batchModels:   models,
```

Comment:
I don't think it's worth having you chance since it's the old UI. But this seems like an odd interface. When batchModels is used it doesn't look like originalModel is used (did I miss it?). And we could've just maintained the old interface and  differentiated between array of models and single model within the component.